ui/static/ui/js/listing/listing.jsx

```diff
@@ -186,6 +191,36 @@ define('listing',
         };
         return React.createElement(ListingResources.ListingPage, options);
       },
+      componentDidUpdate: function () {
```

Comment:
Why does this have to be implemented? It seems like you can move `closeLearningResourcePanel` inside this class and handle the dialog box completely within `closeLearningResourcePanel`